README.md

```diff
@@ -256,6 +256,10 @@ MicrosoftAppPassword           | Microsoft Application Password
 
 Once the bot is deployed and running you can connect different channels to it such as Facebook, Teams, Skype etc using an Azure Bot Channel Registration.  For documentation on that process see:  https://docs.microsoft.com/en-us/bot-framework/bot-service-quickstart-registration
 
+## Deploy Conversation Learner Sample container to Azure
+You can deploy the conversation learner sample to Azure and quickly get started with your Conversation Learner model training
```

Comment:
You can now deploy Conversation Learner Samples to Azure as a docker container to start trying out the UI without having to setup a local development environment.

Reply:
Hmm, I also might add a warning about this is not safe for production use. I worry people will start with this, then add it to a channel, and then try forgot about security and just keep moving forward and let users try it.

I'm not sure if the configuration of directline prevents use in channels, but either way they could have malicious user find the path to the UI and manipulate their model.